bokeh/util/tests/test_callback_manager.py

```diff
@@ -192,7 +192,7 @@ def test_trigger():
     assert good.last_new == 43
 
 def test_trigger_with_two_callbacks():
-    m = cbm.CallbackManager()
+    m = cbm.PropertyCallbackManager()
     good1 = _Good()
     good2 = _Good()
     m.on_change('foo', good1.method)
```

Comment:
once there is checking for event callbacks, will need similar set of tests for event callbacks as well.